website/static/js/mithrilHelpers.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+var $ = require('jquery');  // jQuery
+
+/* Send with ajax calls to work with api2 */
+var apiV2Config = function (xhr) {
+    xhr.withCredentials = window.contextVars.isOnRootDomain;
```

Comment:
We want to avoid using `window` in reuable modules like this. 

See my suggested API for this function here: https://github.com/CenterForOpenScience/osf.io/pull/5825#discussion_r67696208 .